Appraisals

```diff
@@ -9,3 +9,11 @@ appraise 'rails-7.0' do
   gem 'activerecord', '~> 7.0.2'
   gem 'activesupport', '~> 7.0.2'
 end
+
+appraise 'rails-7.1' do
+  gem 'activerecord', '~> 7.1.1'
+  gem 'activesupport', '~> 7.1.1'
+
+  # Rails 7.1 support was added in 4.1.8
+  gem 'delayed_job_active_record', '>= 4.1.8'
```

Comment:
It felt odd to add this as a dependency on this gem directly, but doing it here at least forces the test suite to use the minimum required. Prior to this, the bundler cache was happily providing 4.1.7 which didn't work. If you prefer I can just bust the bundler cache in CI.

Reply:
It's a painless upgrade so I think it's fine to just update the gemspec to require delayed_job_active_record >= 4.1.8.